README.md

```diff
@@ -1,70 +1,22 @@
-# Getting Started with Create React App
+# Weather app with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project is built with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## Concepts learned through this excercise
 
-In the project directory, you can run:
+## React
```

Comment:
This should be a list really, at present it looks like a series of headings for a readme without any content